cognite/client/data_classes/data_modeling/instances.py

```diff
@@ -334,7 +334,7 @@ def to_pandas(  # type: ignore [override]
         pd = local_import("pandas")
         col = df.squeeze()
         prop_df = pd.json_normalize(col.pop("properties"), max_level=2)
-        if remove_property_prefix:
+        if remove_property_prefix and not prop_df.empty:
```

Comment:
After adding all tests, could you run them with the lowest pandas version we support, just in case?